### PR TITLE
Paste: add table support

### PR DESCRIPTION
**Pull request by ellatrix**
## Description
* Adds support for tables.
* Removes block level content in wrappers that can only allow inline content.
* Adds integration tests for table and separator where applicable.
* Fixes a bug in the isPlain check for HTML, adds tests.
* Adds a readme file that includes a support table for editors.

### CODE REVIEW

File: blocks/api/paste/utils.js [hunk 1]
```diff
@@ -125,10 +148,10 @@ export function isPlain( HTML ) {
 	doc.body.normalize();
 
 	// If it's plain text, there should only be one node left.
-	return doc.body.childNodes.length === 1;
+	return doc.body.childNodes.length === 1 && doc.body.firstChild.nodeType === TEXT_NODE;
 }
 
-function deepFilterHelper( nodeList, filters, doc ) {
+export function deepFilterHelper( nodeList, filters, doc ) {
```
Comment (georgeh) [inline blocks/api/paste/utils.js]: I was worried that a helper was being exported solely for testing, but I see it's used in `inline-content-converter.js`. Is there a more descriptive name for this, to differentiate with `deepFilter` (or maybe renaming both)?

Comment (iseulde) [reply blocks/api/paste/utils.js]: Sure :)

**Comment by iseulde** [comment]
@georgeh Is this looking good to merge otherwise?

**Review by georgeh — APPROVED** [review]
:shipit:

**Pull request merged**